Source/Moq.Analyzers.Test/PackageTests.Baseline#manifest.verified.nuspec

```diff
@@ -0,0 +1,17 @@
+﻿<package>
+  <metadata>
+    <id>Moq.Analyzers</id>
+    <version>********</version>
+    <authors>Andrey "Litee" Lipatkin</authors>
+    <licenseUrl>https://github.com/Litee/moq.analyzers/blob/master/LICENSE</licenseUrl>
```

Comment:
We should switch from the old `<licenseUrl>` to the new `<licenseExpression>`. Tracked by #31.